appendixes/mysite/tests/behat/features/bootstrap/Context/FeatureContext.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Mysite\Test\Behaviour;
+
+use SilverStripe\BehatExtension\Context\SilverStripeContext,
+    SilverStripe\BehatExtension\Context\BasicContext,
+    SilverStripe\BehatExtension\Context\LoginContext,
+    SilverStripe\BehatExtension\Context\FixtureContext,
+    SilverStripe\Framework\Test\Behaviour\CmsFormsContext,
+    SilverStripe\Framework\Test\Behaviour\CmsUiContext,
+    SilverStripe\Cms\Test\Behaviour;
+
+// PHPUnit
+require_once 'PHPUnit/Autoload.php';
+require_once 'PHPUnit/Framework/Assert/Functions.php';
+
+/**
+ * Features context
+ *
+ * Context automatically loaded by Behat.
+ * Uses subcontexts to extend functionality.
+ */
+class FeatureContext extends SilverStripeContext {
```

Comment:
This file will get overwritten by the `behat --init` in your shell script, see https://github.com/jeffreyguo/silverstripe-behat-extension/blob/master/src/SilverStripe/BehatExtension/Console/Processor/InitProcessor.php. Which is what I was referring to in my earlier comments: FIles should be copied by that `InitProcessor.php`, not a shell script.